openedx/core/djangoapps/content/learning_sequences/views.py

```diff
@@ -32,9 +30,6 @@ class CourseOutlineView(APIView):
     # We want to eventually allow unauthenticated users to use this as well...
     authentication_classes = (JwtAuthentication, SessionAuthenticationAllowInactiveUser)
 
-    # For early testing, restrict this to only global staff...
-    permission_classes = (IsStaff,)
```

Comment:
Can we leave this in for now, and use the masquerading feature (the querystring parsed out by `_determine_user`) to test whether the enrollment rules are being applied properly?

Reply:
ofcourse